go/vt/sqlparser/sql.y

```diff
@@ -2331,6 +2370,7 @@ reserved_keyword:
 | LOCALTIMESTAMP
 | LOCK
 | MATCH
+| MAXVALUE
```

Comment:
Did you try these 2 values in `non_reserved_keyword` first? Only put them in `reserved_keyword` if they cause a conflict, otherwise we greatly prefer non_reserved. Just checking.